src/es_archiver/lib/indices/generate_index_records_stream.js

```diff
@@ -38,19 +38,22 @@ export function createGenerateIndexRecordsStream(client, stats) {
           ]
         });
 
-        const { [index]: { aliases } } = await client.indices.getAlias({ index });
-        const { settings, mappings } = resp[index];
+        for (const [index, { settings, mappings }] of Object.entries(resp)) {
+          const { [index]: { aliases } } = await client.indices.getAlias({ index });
 
-        stats.archivedIndex(index, { settings, mappings });
-        callback(null, {
-          type: 'index',
-          value: {
-            index,
-            settings,
-            mappings,
-            aliases,
-          }
-        });
+          stats.archivedIndex(index, { settings, mappings });
+          this.push({
+            type: 'index',
+            value: {
+              index: index.startsWith('.kibana') ? '.kibana_1' : index,
```

Comment:
Pretty close to a dupe of the logic in generate_doc_records_stream. I wonder if it should be consolidated. I'm also fine with the duplication, as it's only 2 places and fairly minor.